src/builder.rs

```diff
@@ -305,36 +283,34 @@ impl ClientBuilder {
     }
 
     /// Build the Client instance.
-    pub fn finish(mut self) -> Result<Client> {
-        // Add default nodes
-        if !self.offline {
-            self.node_manager_builder = self.node_manager_builder.add_default_nodes(&self.network_info)?;
-            // Return error if we don't have a node
-            if self.node_manager_builder.nodes.is_empty() && self.node_manager_builder.primary_node.is_none() {
-                return Err(Error::MissingParameter("Node"));
-            }
-        }
+    pub fn finish(self) -> Result<Client> {
         let network_info = Arc::new(RwLock::new(self.network_info));
-        let nodes = self
-            .node_manager_builder
-            .nodes
-            .iter()
-            .map(|node| node.clone().into())
-            .collect();
+
         #[cfg(target_family = "wasm")]
-        let (sync, network_info) = (Arc::new(RwLock::new(nodes)), network_info);
+        let (sync, network_info) = (Arc::new(RwLock::new(HashSet::new())), network_info);
```

Comment:
Sorry I broke some things on wasm
```suggestion
        let (healthy_nodes, network_info) = (Arc::new(RwLock::new(HashSet::new())), network_info);
```